x-pack/plugins/cases/server/client/cases/update.ts

```diff
@@ -337,12 +350,58 @@ async function updateAlerts({
   await casesClientInternal.alerts.updateStatus({ alerts: alertsToUpdate });
 }
 
+function partitionPatchRequest(
+  casesMap: Map<string, SavedObject<ESCaseAttributes>>,
+  patchReqCases: CasePatchRequest[]
+): {
+  nonExistingCases: CasePatchRequest[];
+  conflictedCases: CasePatchRequest[];
+  casesToAuthorize: Array<SavedObject<ESCaseAttributes>>;
+} {
+  const reqCasesMap = patchReqCases.reduce((acc, req) => {
```

Comment:
Instead of creating the map, could you just populate `nonExistingCases`, `conflictedCases`, and `casesToAuthorize` in a for loop here? Guessing you created the map because you could have duplicate requests and you only want the latest? If that's the case, you could just loop over the array backwards and skip any id's you've already seen?

Reply:
Yep, no idea why I created a map haha